helpers/setup.sh

```diff
@@ -137,8 +137,10 @@ gcloud services enable \
 
 if [[ "$IS_UPDATE" == "0" ]]; then
 
-    # If is an update i don't create service accout and re-download credentials
+    # Create new service accout and download credentials
     echo "Creating a new service account ${SERVICE_ACCOUNT_EMAIL} ..."
+    export FORSETI_SETUP_SERVICE_ACCOUNT_NAME="${SERVICE_ACCOUNT_NAME}"
```

Comment:
Is this variable used elsewhere?

Reply:
It's used by the cleanup script, it was also removed by the PR mentioned above.